src/xcode/ENA/ENA/Source/HTTPClient/Security/EvaluateTrust/DynamicEvaluateTrust.swift

```diff
@@ -38,9 +38,14 @@ struct DynamicEvaluateTrust: EvaluateTrust {
 			completionHandler(.useCredential, URLCredential(trust: trust))
 		case .failure(let error):
 			Log.debug("AuthenticationChallenge failed with error \(error.localizedDescription)", log: .client)
+			trustEvaluationError = error
```

Comment:
When will is set to nil? Otherwise it might block all future requests?

Reply:
Good catch!!! 
Reseting it now after the completion call.